src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```
@@ -3577,4 +3570,10 @@ private function getPublicDirectory(ContainerBuilder $container): string
 
         return isset($composerConfig['extra']['public-dir']) ? $projectDir.'/'.$composerConfig['extra']['public-dir'] : $defaultPublicDir;
     }
+
+    private function registerObjectMapperConfiguration(ContainerBuilder $container, PhpFileLoader $loader): void
```

let's move the method up next to the other register*
also, let's add a class_exists check to throw when the component is not found, see registerJsonStreamerConfiguration for inspiration